chapter_4/ex04_directory/CMakeLists.txt

```
@@ -0,0 +1,14 @@
+# CMakeLists file for the Chapter 4 Example nr. 04: 
+# Install directories
+#
+# SPDX-License-Identifier: MIT
+
+project(
+  ch4_ex04_directory
+  VERSION 1.0
+  DESCRIPTION "Chapter 4 Example 04, install directory"
+  LANGUAGES CXX)
```

This project doesn't build anything. A pointless toolchain booting process can be avoided with this:

```suggestion
  LANGUAGES NONE)
```

---

Good catch. I'll apply the changes.